trade-finance-manager-ui/test-helpers/fee-record-payment-group-item.ts

```diff
@@ -23,4 +23,6 @@ export const aFeeRecordPaymentGroup = (): FeeRecordPaymentGroup => ({
   paymentsReceived: null,
   totalPaymentsReceived: null,
   status: 'TO_DO',
+  reconciledByUser: undefined,
+  dateReconciled: undefined,
```

Comment:
Can we update the type so that these properties are optional, rather than having to declare undefined values?

Reply:
These fields are optional - declaring them as `undefined` simply makes the test helper more explicit

Reply:
I think it's wasted lines of code, declaring things as undefined, there's no need or benefit to it IMO.

Reply:
The benefit is readability and clarity. I will remove for the sake of getting this PR merged